Say Their Names/Source/Controller/Home/HomeView.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  HomeView.swift
+//  Say Their Names
+//
+//  Created by Joseph A. Wardell on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class HomeView: UIView {
+
+    var customNavBar: UIView!
+    var locationCollectionView: UICollectionView!
+    var peopleCollectionView: UICollectionView!
+    var searchButton: UIButton!
+    var separator : UIView!
```

Comment:
```suggestion
    var separatorView : UIView!
```

Reply:
Typically, if I see the suffix "View" at the end of a property name, it suggests to me that the property lets me view that thing (e.g. tableView, personView, etc.).  This is just a separator meant to visually denote where one collection view ends and the next begins.  I think separator is the right name.